docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -57,13 +57,12 @@ Arrays are not supported.
 [[dfa-regression-supervised]]
 ==== Training the {regression} model
 
-{regression-cap} is a supervised machine learning process, which means that you 
-need to supply a labeled training dataset that has some {feature-vars} and a 
-{depvar}. The {regression} algorithm learns the relationships between the 
-feature and the {depvar}. Once you've trained the model on your training 
-dataset, you can reuse the knowledge that the model has learned about the 
-relationships between the data points to make certain inferences about new data 
-points.
+{regression-cap} is a supervised {ml} process, which means that you need to 
+supply a labeled training dataset that has some {feature-vars} and a {depvar}. 
+The {regression} algorithm learns the relationships between the feature and the 
+{depvar}. Once you've trained the model on your training dataset, you can reuse 
+the knowledge that the model has learned about the relationships between the 
```

Comment:
IMO, it's logically incorrect to speak about the relationship between data points in this sentence, since in the previous sentence we were saying that the relationship is between the features and the dependent variable.